### PR TITLE
Add support for those distros which don't provide 'adduser'

### PR DESCRIPTION
**Pull request by alessio**
Comments on this would be appreciated.

Thanks for considering.

### CODE REVIEW

File: sshcommand [hunk 1]
```diff
@@ -4,6 +4,30 @@ set -e
...
+f_adduser() {
+  local l_user l_platform
+
+  l_user=$1
+  l_platform="$(f_print_os_name)"
+  case $l_platform in
+    Debian|Ubuntu)
+      adduser --disabled-password --gecos "" $l_user
+      ;;
+    SLES|*)
```
Comment (josegonzalez) [inline sshcommand]: Can this be done on suse as well?

Comment (alessio) [inline sshcommand]: SLES is SUSE Linux Enterprise Server so yes, it works.
Actually all of it is part of a porting to SUSE platforms.

Comment (josegonzalez) [inline sshcommand]: What I meant by my question is whether it works on the non-enterprise version, like opensuse.

Comment (alessio) [inline sshcommand]: Yes, it works on OpenSUSE and SLES as both have the same useradd implementation. I tested this on both too.

Comment (jvanbaarsen) [inline sshcommand]: What is the platform name that OpenSUSE will output for `f_print_os_name`? Or is that what the `*` is for? If the latter is true, what about removing the `SLES` and just use `*` as a fallback?

Comment (alessio) [inline sshcommand]: I'm OK with that :+1:

**Comment by josegonzalez** [comment]
@alessio mind updating the PR? We can merge after :)

**Comment by alessio** [comment]
Will do shortly, thanks

**Comment by alessio** [comment]
Rebased and ready for the merge.
Thanks for considering.

**Pull request merged**